### PR TITLE
OAK-8910 - Improve OAK Lucene Index Documentation

### PR DESCRIPTION
**Pull request by averma21**

### CODE REVIEW

File: oak-doc/src/site/markdown/query/indexing.md [hunk 1]
```diff
@@ -426,6 +426,21 @@ NRT indexing expose a few configuration options as part of the [LuceneIndexProvi
   to hold Lucene documents for indexing in the `nrt` mode. 
   The default size is 10000.
 
+## <a name="superseding"></a> Superseding an index
```
Comment (thomasmueller) [inline oak-doc/src/site/markdown/query/indexing.md]: This page currently uses https://en.wikipedia.org/wiki/Capitalization#Title_case so in theory you should use "Superseding an Index". I would do that for consistency. However, I know for most people this isn't important. It's up to you.

**Review by thomasmueller — APPROVED** [review]

**Comment by thomasmueller** [comment]
Let me know if you want me to commit the change.

File: oak-doc/src/site/markdown/query/indexing.md (hunk 1 above)
Comment (averma21) [reply oak-doc/src/site/markdown/query/indexing.md]: No issues. I have changed it.

**Comment by averma21** [comment]
> Let me know if you want me to commit the change.


To svn? Yes, please. I have added the patch to the issue as well.
https://issues.apache.org/jira/browse/OAK-8910

**Review by fabriziofortino — APPROVED** [review]
+1 LGTM 

What about adding a link to the Lucene common analyzers https://lucene.apache.org/core/4_7_1/analyzers-common/overview-summary.html? I would add it to the section `Specify analyzer class directly`

**Pull request closed without merging**